test/unit/auth/token-verifier.spec.ts

```diff
@@ -309,6 +326,34 @@ describe('FirebaseTokenVerifier', () => {
       }).to.throw('First argument to verifyIdToken() must be a Firebase ID token');
     });
 
+    const errorTypes = [
```

Comment:
Why use an array with just one element?

Reply:
I was hoping that we will have other error types to include here in the future. Will remove the array for now and we can add it later when we have more if needed.